src/core/Dropdown/Dropdown.test.tsx

```diff
@@ -50,6 +50,16 @@ describe('Basic dropdown', () => {
     expect(input).toHaveAttribute('name', 'dropdown-test');
   });
 
+  it('should have disabled styles when disabled', async () => {
+    const { findByRole } = render(
+      <Dropdown labelText="Dropdown" disabled>
+        <Dropdown.item value={'item-1'}>Item 1</Dropdown.item>
+      </Dropdown>,
+    );
+    const button = await findByRole('button');
```

Comment:
Are these queries really async by nature? Or is this the infamous useLayoutEffect etc. error message thing?

Reply:
To be honest, I just saw all the other tests being written with the async and followed suit 😅

Reply:
This is probably the problematic case that causes a lot of warnings when executing, so maybe better to leave as is.